examples/complete/main.tf

```diff
@@ -17,6 +17,17 @@ module "key_protect" {
   }
 }
 
+module "event_notification" {
+  source            = "terraform-ibm-modules/event-notifications/ibm"
+  version           = "1.0.4"
+  resource_group_id = module.resource_group.resource_group_id
+  name              = "${var.prefix}-en"
+  tags              = var.resource_tags
+  plan              = "lite"
```

Comment:
what are the limitations around lite plan? Are you only allowed 1 per account? 1 per resource group? 
If its 1 per account, we can't use it, as tests will clash. If it is 1 per resource group, we will need to update the tests to create a new resource group every time (aka comment out `ResourceGroup:      resourceGroup` in setup options)

Reply:
There is no limitation per resource group.
The lite plan is for limited number of notifications.
```Lite: This plan gives you unlimited ingested events, 10 topics, 2 filters per topic, 5 destinations, 20 outbound emails, 20 outbound SMSes, 20 outbound webhooks, and 1000 notifications per push destination. 10 subscriptions are allowed, and a subscription can have a maximum of 3 email recipients.```